packages/components-css/icon-css/src/_mixin.scss

```diff
@@ -1,21 +1,46 @@
 @mixin nl-icon {
-  block-size: var(--nl-icon-block-size, 1em);
+  // Default to 1lh × 1lh
+  --_nl-icon-block-size: 1lh;
+  --_nl-icon-inline-size: 1lh;
+
+  align-items: center;
+
+  // background-color: color-mix(in srgb, tomato 25%, transparent);
+  block-size: var(--nl-icon-block-size, var(--_nl-icon-block-size));
   color: var(--nl-icon-color, currentcolor);
-  display: inline-block;
+  display: inline-flex;
   font-size: var(--nl-icon-font-size, 1em);
-  inline-size: var(--nl-icon-inline-size, 1em);
+  inline-size: var(--nl-icon-inline-size, var(--_nl-icon-inline-size));
   inset-block-start: var(--nl-icon-inset-block-start, 0);
+  justify-content: center;
+  min-block-size: var(--nl-icon-block-size, var(--_nl-icon-block-size));
+  min-inline-size: var(--nl-icon-inline-size, var(--_nl-icon-inline-size));
   pointer-events: none;
   position: relative;
+  vertical-align: middle;
 }
 
-@mixin nl-icon__svg {
-  /* stylelint-disable-next-line property-disallowed-list */
-  height: 100%;
-  /* stylelint-disable-next-line property-disallowed-list */
-  width: 100%;
+@mixin nl-icon--forced-colors {
+  color: CanvasText;
+}
```

Comment:
`CanvasText` in plaats van `currentColor` heeft onbedoelde gevolgen. De bedoeling is dat een icon de kleur overneemt van de context, bijvoorbeeld `LinkText` in een link, `GrayText` in een disabled button, etcetera.

> De Icon component is altijd zichtbaar in “forced colors” modus.

Waarschijnlijk is dit succescriterium in https://github.com/nl-design-system/kernteam/issues/1488 dus niet compleet en expliciet genoeg.